packages/plan-form/src/Form/index.tsx

```diff
@@ -192,6 +192,14 @@ const PlanForm = (props: PlanFormProps) => {
 
   const [form] = Form.useForm();
 
+  // if active date range has no end date (end date === null)
+  // convert end date (null) to Moment type (moment.Moment | undefined)
+  useEffect(() => {
+    if (!initialValues.dateRange[1]?.isValid()) {
+      initialValues.dateRange[1] = undefined;
+    }
```

Comment:
should we make this change here: https://github.com/OpenSRP/web/blob/e9476a549f58fee7f42dbac339a63dc8beebc369/packages/plan-form/src/helpers/utils.tsx#L112 i.e. at the point where the initial values are generated. 

Though i am a not sure this is a problem we should consider solving from here, given that the kind of plan that causes this issue is a plan that does not honor the contract of how plans created on the web look like. i think the better solution is to disable the plans views for instances that use a single global-plan that has its own schema that would potentially break the plan-form. @moshthepitt thoughts?

Reply:
> better solution is to disable the plans views for instances that use a single global-plan that has its own schema

@p-netm didn't get what you meant here.

But you bring up a good point.  AFAIK all plans need to have an end date for them to be valid.  What do you think we should do when we encounter an invalid plan - either via a missing end date like here, or in another way?

Reply:
disable the plans module. the plan in question is as described here https://docs.google.com/document/d/1mJSwmxl82giN3oXIvUNj3YHFDfr__QZdFjb2X2rpiQ0/edit#heading=h.uir972vohp2g, that is app instances where the assignment flow is not "plan-based", so we have a global plan created outside the web to facilitate things like assignments.

Reply:
@moshthepitt

Reply:
> Though i am a not sure this is a problem we should consider solving from here, given that the kind of plan that causes this issue is a plan that does not honor the contract of how plans created on the web look like. i think the better solution is to disable the plans views for instances that use a single global-plan that has its own schema that would potentially break the plan-form. @moshthepitt thoughts?

I was of a similar opinion, the UI doesn't seem to allow you create a plan without an end date to start with. Is this seed data?